src/StreamJsonRpc.Shared/DataContracts/JsonRpcMessage.cs

```diff
@@ -120,6 +127,13 @@ public object[] GetParameters(ParameterInfo[] parameterInfos, JsonSerializer jso
                 return this.Parameters.ToObject<object[]>();
             }
 
+            if (parameterInfos.Length == 1 && parameterInfos[0].ParameterType == typeof(JToken))
+            {
+                // We want to ignore methods that only have one parameter and the parameter type is JToken.
+                // This is reserved to be called when the parameter is passed as an object.
+                return null;
```

Comment:
Talked to Ilya about this and it looks like that's just the contract for using streamsjsonrpc.  The server shouldn't have 2 methods both with one argument and one of them is JToken.  JToken is used as a catch all.  I'll modify my unit test to not break the contract and also have a unit test in place to verify the contract.